setup.py

```diff
@@ -26,11 +33,13 @@
         "Topic :: Scientific/Engineering",
         "Topic :: System :: Distributed Computing",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
     ],
     keywords="dask geopandas spatial distributed cluster",
-    description="GeoPandas objects backed with Dask",
+    description="Parallel GeoPandas with Dask",
```

Comment:
I updated this with the "tagline" we have in the README / github description. Do we find this a good one? Can also update it everywhere to yet something else (I am wondering if "parallel" is clear enough that it is also "distributed")

Reply:
Dask itself uses `parallel` primarily, so I am fine with that.